### DISCUSSION TITLE
Develop as a web app in the browser

### DISCUSSION THREAD

**Discussion by vonWolfehaus** [cawa-93/vite-electron-builder#895] [Q&A] [answered]
This is a really great template, thank you! I have an electron app running, but now I want to run that app in the browser. I've made sure to separate any electron-specific node stuff so that it can be run in the browser (it passes jsdom tests just fine, ie without node).

I tried running `vite` but it can't find App.vue or any of the components it uses. Any idea how to get it to serve out of the renderer package? Or does it need to use the /dist folder? I'm a bit lost.

**Comment by cawa-93** [comment, marked as answer]
Here's web app: https://github.com/cawa-93/vite-electron-builder/tree/main/packages/renderer
App.vue is here: https://github.com/cawa-93/vite-electron-builder/blob/main/packages/renderer/src/App.vue
You can use all vite cli inside https://github.com/cawa-93/vite-electron-builder/tree/main/packages/renderer like [`build:renderer`](https://github.com/cawa-93/vite-electron-builder/blob/25e987e86127b9ae068a4c7ea3e936a3c00226f9/package.json#L16) script does.

**Reply by vonWolfehaus** [reply]
I didn't think about using the `build:renderer` script like that, but it worked. Thanks for the tip!
